src/azul/service/manifest_controller.py

```diff
@@ -137,28 +140,45 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            unix_cmd = f'curl "{location}"'
+            windows_cmd = f'curl.exe "{location}"'
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
+            except KeyError:
                 url = self.manifest_url_func(fetch=False,
                                              catalog=manifest.catalog,
                                              format_=manifest.format_,
                                              filters=json.dumps(manifest.filters),
                                              objectKey=manifest.object_key)
             else:
                 url = manifest.location
+            command_lines = self.service.command_lines(manifest, url)
+            unix_cmd = command_lines['bash']
+            windows_cmd = command_lines['cmd.exe']
             body = {
                 'Status': 302,
                 'Location': url,
-                'CommandLine': self.service.command_lines(manifest, url)
+                'CommandLine': command_lines
             }
-
+        instructions = self._curl_instructions({'Unix': unix_cmd,
+                                                'Windows': windows_cmd})
         if fetch:
             return Response(body=body)
         else:
             headers = {k: str(body[k]) for k in body.keys() & {'Location', 'Retry-After'}}
-            return Response(body='', status_code=body['Status'], headers=headers)
+            return Response(body=instructions, status_code=body['Status'], headers=headers)
+
+    def _curl_instructions(self, cmds_by_os: Mapping[str, str]) -> str:
```

Comment:
Why is this a method? I only see one call.

Reply:
>Why is this a method?
I thought it looked neater that way.